src/MemberReportExtension.php

```diff
@@ -11,6 +12,16 @@
  */
 class MemberReportExtension extends DataExtension
 {
+
+    /**
+     * Connect a link to LoginAttempt
+     * 
+     * @var array
+     * @config
+     */
+    private static $has_many = [
+        'LoginAttempts' => LoginAttempt::class
+    ];
```

Comment:
Are we able to achieve this functionality without introducing this new relationship?

Reply:
It's not really a new relationship, rather a reflection of an existing one:
https://github.com/silverstripe/silverstripe-framework/blob/4.1.0/src/Security/LoginAttempt.php#L46

I thought as `has_many` doesn't impact storage at all that this was an acceptable addition.
It can of course be done without it. Would you like me to refactor?

Reply:
Oh ok, so that relationship is always there, but whether or not it's active is configurable? If that's the case perhaps you could just leave a comment on that line to point that out

Reply:
Sure :)